compiler/rustc_const_eval/src/const_eval/dummy_machine.rs

```diff
@@ -102,7 +102,7 @@ impl<'mir, 'tcx: 'mir> interpret::Machine<'mir, 'tcx> for DummyMachine {
         _destination: &interpret::MPlaceTy<'tcx, Self::Provenance>,
         _target: Option<BasicBlock>,
         _unwind: UnwindAction,
-    ) -> interpret::InterpResult<'tcx> {
+    ) -> interpret::InterpResult<'tcx, Option<ty::Instance<'tcx>>> {
```

Comment:
The `Option<Instance>` scheme is what codegen_ssa and codegen_cranelift both use to compile to a function call in case the intrinsic doesn't have a backend impl